icu4j/main/core/src/main/java/com/ibm/icu/segmenter/Segments.java

```diff
@@ -26,7 +26,7 @@
  *
  * Segments segments = wordSeg.segment("Das 21ste Jahrh. ist das beste.");
  *
- * List<CharSequence> words = segments.subSequences().collect(Collectors.toList());
+ * List&gt;CharSequence> words = segments.subSequences().collect(Collectors.toList());
```

Comment:
```suggestion
 * List&lt;CharSequence&gt; words = segments.subSequences().collect(Collectors.toList());
```

Reply:
It is not required. But sure :-)

Reply:
The important part of the change was on the left side where you had `&gt;` ...